Makefile

```diff
@@ -1,27 +1,40 @@
 CC      = gcc
-C_FLAGS = -Wall -g -pedantic -Werror
+C_FLAGS = -Wall -pedantic -Werror
```

Comment:
Why are you disabling -g? I'd think we'd want debugging information in the binaries so core dumps are readable.